tests/test_name.py

```diff
@@ -432,6 +432,13 @@ def test_parsed_name_initials():
     assert expected == parsed_name.first_initials_list
 
 
+def test_format_author_name_with_initials_when_first_name_is_similar_to_title():
+    expected = "E. Lieber"
+
+    assert expected == format_name(
+        'Lieber, Ed', initials_only=True)
+
+
```

Comment:
how it will behave when somebody will have name like `Lieber, Ed Viktor`?

Reply:
(it would be nice to have more test cases there cause it might mess up a some things in query parser/hep)